modules/common_fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/Enrich.scala

```diff
@@ -55,7 +61,7 @@ object Enrich {
   /** Default adapter registry, can be constructed dynamically in future */
   val adapterRegistry = new AdapterRegistry()
 
-  val processor: Processor = Processor(generated.BuildInfo.name, generated.BuildInfo.version)
+  val processor: Processor = Processor(BuildInfo.name, BuildInfo.version)
```

Comment:
I think we'd get something like `Processor("common-fs2", "2.0.0")`, which is not exactly what we want.